pastas/modelstats.py

```diff
@@ -497,6 +497,34 @@ def diagnostics(
         stats: tuple = (),
         float_fmt: str = "{0:.2f}",
     ) -> DataFrame:
+        """Methods to compute various diagnostics checks for a time series.
+
+        Parameters
+        ----------
+        tmin: str or pandas.Timestamp, optional
+            Start date of the residual time series to compute the diagnostics checks for.
```

Comment:
I'd change this to:
`start date of the residual / noise time series` or just `start date of the time series`